src/index.js

```diff
@@ -0,0 +1,40 @@
+//TEST FORM BUILDER
+var formGen = require('./FormGenerator.js');
+var testInputs =[{
+  type: 'label',
+  name: 'label-name',
```

Comment:
Please make sure all of the relevant optional attributes for the Label input type are handled in `FormGenerator.js`.